www/js/pages/sign-in/sign-in.controller.js

```diff
@@ -83,30 +87,120 @@
       languageService.update(vm.language);
     }
 
-    // change different type of signing methods
-    vm.changeSigninView = function(_isBack){
-      _isBack = _isBack || false;
-
-      $scope.$applyAsync(function(){
-        if(!vm.showSignin && vm.showTokenForm){
-          vm.showTokenForm = false;
-          vm.showSignin = true;
-        }
-        else if(vm.showSignin && !vm.showTokenForm && _isBack){
-          vm.showSignin = false;
+    // sign up email verify
+        vm.verifyUserMail = function() {
+          var mail = vm.data.mail;
+          websocketService.sendRequestFor.accountOpening(mail);
         }
-        else if(vm.showSignin && !vm.showTokenForm){
-          vm.showTokenForm = true;
-          vm.showSignin = false;
+        $scope.$on('verify_email', (e, verify_email) => {
```

Comment:
can't find the problem here, listener is out of function here, but I will search it through the rest of app and correct it, if you're sure about it in this part of code please tell me, I mean the red parts are deleted and listener seems to be in a function here but is not.